src/blueprints/timeline-state-resolver-types/sisyfos-types.ts

```diff
@@ -16,14 +16,16 @@ export interface SisyfosChannelsTimelineObject extends TimelineObject {
   content: {
     deviceType: DeviceType.SISYFOS;
     type: SisyfosType.CHANNELS;
-    channels: ({
-      /** The mapping layer to look up the channel from */
-      mappedLayer: string;
-    } & SisyfosChannelOptions)[];
+    channels: SisyfosChannel[];
     overridePriority?: number;
   }
 }
 
+export interface SisyfosChannel extends SisyfosChannelOptions {
+  /** The mapping layer to look up the channel from */
+  mappedLayer: string
+}
+
```

Comment:
I was using the interface at an earlier stage. That usage since then got removed, but i liked the more clean look of the type for `channels` in this file, so i chose to keep it.

I have for now reverted the change, even though i do think it makes the type more readable, compared to the type you originally made.